.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,43 @@
+name: Prepare Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      next-version:
+        description: 'Override next development version (e.g. 5.0.0-SNAPSHOT). If not set then the version will be calculated by maven release plugin'
+
+jobs:
+  prepare-release:
+    runs-on: ubuntu-latest
+    name: Prepare release
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ssh-key: "${{ secrets.COMMIT_KEY }}"
+      - name: Setup JDK
+        uses: actions/setup-java@v4
+        with:
+          java-version:  17
+          distribution: 'temurin'
+          cache: 'maven'
+          gpg-private-key: ${{ secrets.MAVEN_GPG_PRIVATE_KEY }}
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE
+
+      - name: Prepare release
+        run: |
+          if [ -n "${{ github.event.inputs.next-version }}" ]; then
+              EXTRA_ARGS=-DdevelopmentVersion=${{ github.event.inputs.next-version }}
+          fi
+          git config user.name "GitHub Actions Bot"
```

Comment:
We use `[devOpsHazelcast](https://github.com/hazelcast/hazelcast-docker/commits?author=devOpsHazelcast)' for the name in Hazelcast Platform. I presume its the same user?

Reply:
yeah we can use both, here since it's a github actions workflow that commits, I think this is better devopsHazelcast seems like a jenkins/lab machine user name to me. 

I'm fine by both

Reply:
I prefer `devopsHazelcast` as no space 

The email is tied to the user https://github.com/orgs/hazelcast/people/devOpsHazelcast so I think `devopsHazelcast` makes sense as GitHub will make the connection I think

Not too fussed so will leave it to you

Reply:
ok done, 5ddbd2c

Reply:
thanks